index.js

```
@@ -47,10 +47,23 @@ function getPathFromRegex(regexp) {
+  return [...acc, stack];
+}
+
+function combineExpress5Stacks(acc, stack) {
+  if (stack.handle?.stack) {
+    // FIXME express5 we can't get the router path for nested
+    // The stack object has not reference to parent router path
+    // Tried looking through whole express app object and
+    // couldn't find any reference for nested routes.
+    // For now we just intedcated nested routes with ~
+    const routerPath = stack.path ?? '/~';
+    return [...acc, ...stack.handle?.stack.map((nestedStack) => ({ routerPath, ...nestedStack }))];
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)